rom-hacking/scripting/commands/talkmsg.md

```diff
@@ -13,7 +13,7 @@ Sets PlayTextFeedSe to true if **textSe** is 1.
 ## Syntax
 
 ```c
-_TALKMSG(message, textSe)
+_TALKMSG(message, textSe?)
```

Comment:
Don't shorten words in documentation. Change all instances of `se`

Reply:
@SaltContainer what does the `se` stand for?